### DISCUSSION TITLE
Is Kubelogin compatible with x509 certificate?

### DISCUSSION THREAD

**Discussion by amruta53** [Azure/kubelogin#301] [Q&A]
Incase of kubelogin, Instead of AAD can we use x509 certificate? Since AKS supports x509 certificates for authentication

**Comment by weinong** [comment]
you mean to use x509 client cert to talk to k8s api? yes, you can sign your own CSR and configure the k8s rbac to allow it

**Comment by weinong** [comment]
or do you mean to use x509 client cert to login the AAD service prinicple?
